test/integration/use_int_test.go

```diff
@@ -53,6 +55,36 @@ func (suite *UseIntegrationTestSuite) TestUse() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *UseIntegrationTestSuite) TestReset() {
+	suite.OnlyRunForTags(tagsuite.Use)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("use", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.ExpectExitCode(0)
+
+	python3Exe := filepath.Join(ts.Dirs.DefaultBin, "python3"+osutils.ExeExt)
+	if runtime.GOOS == "windows" {
+		python3Exe = python3Exe + ".bat"
+	}
+	suite.True(fileutils.TargetExists(python3Exe), python3Exe+" not found")
+
+	cp = ts.SpawnWithOpts(e2e.WithArgs("use", "reset"))
+	cp.Expect("Reset default project runtime")
+	cp.Expect("Note you may need to")
+	cp.ExpectExitCode(0)
+
+	suite.False(fileutils.TargetExists(python3Exe), python3Exe+" still exists")
```

Comment:
Can this test also cover the environment cleanup?

Reply:
Yep, at least on Linux and macOS where we have RC files that change the environment.